### ISSUE TITLE
bravery settings

### ISSUE THREAD

**Issue by bridiver** [brave/browser-ios#148]
bravery settings are not consistent with browser-laptop. You should be able to disable ad replacement without disabling ad blocking. Current laptop settings:
replace ads/block ads/allow ads multi-select
block 3rd-party cookies on/off
https everywhere on/off

**Comment by garvankeeley** [comment]
Agreed, but I think it should be a separate setting. I see the use case as user who doesn't want placeholders ads. The brave button and related settings are for site problem use cases.

**Comment by bridiver** [comment]
this is probably a @bradleyrichter call, but I think we should try to make the settings as similar as possible across platforms
